introduction.md

```diff
@@ -1 +1,3 @@
-Hi my name is Lukas with course ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?